template-helpers/breach-detail.js

```diff
@@ -42,6 +42,10 @@ function breachCategory(args) {
 // Big TODO : How should we assign weight to these data types?
 
 const priorityDataClasses = {
+  "social-security-numbers" : {
```

Comment:
There are a 3 breaches with social security numbers in them. But the other 2 have data class "Government issued IDs". Can we make the same priority & glyph apply to that data class too?

Reply:
https://fx-breach-alerts.herokuapp.com/breach-details/Flashback, https://fx-breach-alerts.herokuapp.com/breach-details/GoldSilver, and https://fx-breach-alerts.herokuapp.com/breach-details/Vodafone (see also https://github.com/mozilla/blurts-server/issues/1005)

Reply:
@groovecoder updated! And fixed two spelling errors also.